facilitator/src/bin/facilitator.rs

```diff
@@ -97,6 +87,57 @@ Keys: All keys are P-256. Public keys are base64-encoded DER SPKI. Private keys
 base64 encoded format expected by libprio-rs, or base64-encoded PKCS#8, as documented. \
 ";
 
+/// The string "-input" or "-output", for appending to arg names.
+enum InOut {
+    Input,
+    Output,
+}
+
+impl InOut {
+    fn str(&self) -> &'static str {
+        match self {
+            InOut::Input => "-input",
+            InOut::Output => "-output",
+        }
+    }
+}
+
+/// One of the organizations participating in the Prio system.
+enum Entity {
+    Ingestor,
+    Peer,
+    Own,
+    // Hack: the aggregation bucket is part of the "Own" entity (i.e. whoever is running this).
+    // For the purpose of command line flags it acts as a separate entity.
+    Aggregation,
```

Comment:
Given that we haven't yet broken the second map step out of the aggregation reduce step (#26), does it make sense to separate `Aggregation` from `Portal` yet?